mikoba/ViewModels/Pages/WalletPageViewModel.cs

```diff
@@ -247,7 +245,7 @@ public ICommand RecoverWalletCommand
         {
             get => new Command(async () =>
             {
-                await _dialogService.ShowAlertAsync("Not implemented!", "Notice", "OK");
+                await _dialogService.AlertAsync("Not implemented!", "Notice", "OK");
```

Comment:
Just a comment. We should probably have a dedicated method that just shows `Not Implemented`.